README.md

```diff
@@ -294,3 +304,58 @@ class ArrayHelper_CXFileUniqueID_data_level0 {
   2. Run `dart build_test_dylib.dart`.
 
 Run tests from the root of the package with `pub run test`.
+
+## FAQ
+<details><summary>Removing underscores or renaming declarations.</summary>
```

Comment:
Maybe just use headers instead of details/summary.

Reply:
I think it might be better to use collapsible details/summary for this because maybe this list can grow a lot over time.

Reply:
It's the bottom of the page, so I think it's fine.

Also, if people use ctrl+f to search for something, how does that work if they search for a keyword that is in the collapsed part?

Reply:
- removed details/summary

Search doesn't work for collapsed section.